src/library/dao/playlistdao.cpp

```diff
@@ -414,6 +414,37 @@ void PlaylistDAO::removeHiddenTracks(const int playlistId) {
     emit(changed(playlistId));
 }
 
+void PlaylistDAO::removeEmptyPlaylists(const int hidden) {
+    ScopedTransaction transaction(m_database);
+    // This query deletes all tracks marked as deleted and all
+    // phantom track_ids with no match in the library table
+    QString queryString = QString(
+        "SELECT id FROM Playlists "
+        "WHERE hidden=%1 AND "
```

Comment:
General rule: Prefer parameterized queries over query string formatting. Usually faster and MUCH safer -> SQL injection. Doesn't matter here, but we should get rid of this naive query parameterization style that can be found in many places in the Mixxx code base.